pkg/services/live/features/stream.go

```diff
@@ -0,0 +1,138 @@
+package features
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+type StreamManager struct {
```

Comment:
Nit. Would be nice with some tests of the stream manager. No merge blocker

Reply:
Yep, done!